examples/bridge-app/bridge-common/bridge-app.matter

```diff
@@ -212,10 +212,6 @@ client cluster Binding = 30 {
     fabric_idx fabricIndex = 254;
   }
 
-  attribute TargetStruct binding[] = 0;
-  readonly attribute command_id generatedCommandList[] = 65528;
-  readonly attribute command_id acceptedCommandList[] = 65529;
-  readonly attribute attrib_id attributeList[] = 65531;
```

Comment:
Were removed because as per the zap file. The binding server cluster was not enabled and therefore this should be be generated here as well.